docs/install/docker/windows-docker-ce.md

```diff
@@ -0,0 +1,149 @@
+---
+navTitle: Docker Engine on Windows
+meta:
+   description: Install Docker Engine (Docker CE) on Windows.
+   tags: 
+       - docker for windows
+       - install docker on windows
+       - install docker on windows with wsl2
+       - running docker on windows 11
+---
+
+# How to Install Docker Engine (Docker CE) on Windows using WSL2
+
+## Disclaimer 
+
+Docker containers are inherently a Linux technology. The setup described here for running Docker on Windows using WSL2 is intended for development, testing, or learning purposes and may not be suitable for production environments.
+
+## Prerequisites
+
+Ensure your system meets the following requirements:
+
+- Windows Server 2019 (version 1709) and later
```

Comment:
@ppawlowski 

WSL2 is not available on Server 2019 and non updated systems (only WSL)

Rather than listing out all the ways it can be installed, i would recommend a generic method and min OS version + a quick check.

e.g ask the user to run `wsl --status` to confirm that WSL2 is being used.

```
❯ wsl --status
Default Distribution: Ubuntu
Default Version: 2
```